CHANGELOG.md

```diff
@@ -5,7 +5,18 @@ This is a record of all notable changes to the "omt-odt-language" extension.
 
 Check [Keep a Changelog](http://keepachangelog.com/) for recommendations on
 how to structure this file.
+## [2.4.0]
```

Comment:
Please always surround every header with a line of whitespace on either side. You can use the markdownlint extension in VSCode to check your markdown files.